xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp

```diff
@@ -28,8 +28,8 @@
 #include "utils/StringUtils.h"
 
 #define CONTROL_HEADER_LABEL   1
-#define CONTROL_NONE_AVAILABLE 4
```

Comment:
Yeah sorry, oversight on my part, I think that change was agreed on somewhere in Slack, and was intended to avoid confusion in case users get an empty dialog.